lib/tasks/server/livereload-server.js

```diff
@@ -40,7 +40,7 @@ module.exports = Task.extend({
   start: function(options) {
     var tlroptions = {};
     tlroptions.ssl = options.ssl || false;
-    tlroptions.host = '0.0.0.0';
+    tlroptions.host = options.liveReloadHost || '0.0.0.0';
```

Comment:
commands/serve.js?  there are defaults there already. I just followed the pattern i already saw.